### DISCUSSION TITLE
Is it possible to have multi select filters without needing to use the Apply and Clear buttons?

### DISCUSSION THREAD

**Discussion by sanctus1991** [microsoft-search/pnp-modern-search#3928] [Q&A]
Hi,

As the title says, a customer has a few multi select filters but doesn't want to use the Apply and Clear buttons (just wants the results to filter instantly when a value is clicked).

**Comment by kasperbolarsen** [comment]
That would require a significant amount of development,  but if you are up to the challenge,  we would welcome a PR
